wagtail/contrib/redirects/models.py

```diff
@@ -27,6 +29,25 @@ class Redirect(models.Model):
         on_delete=models.CASCADE
     )
     redirect_link = models.URLField(verbose_name=_("redirect to any URL"), blank=True, max_length=255)
+    automatically_created = models.BooleanField(
+        verbose_name=_("automatically created"),
+        default=False,
+        editable=False,
+    )
+    created_at = models.DateTimeField(verbose_name=_("created at"), auto_now=True, null=True)
+    trigger_content_type = models.ForeignKey(
+        ContentType,
+        models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name='+',
+    )
+    trigger_id = models.PositiveIntegerField(
+        blank=True,
+        null=True,
+        db_index=True,
+    )
+    trigger = GenericForeignKey("trigger_content_type", "trigger_id")
```

Comment:
It looks like this is a foreign key to the action log that created the redirect. It doesn't appear to get used anywhere though. Do we need to capture this?

(If we're not entirely sure we need this, I think we should leave it out and add it back in later if the need arises)

Reply:
Yes, the feature really needs a bit of UX input to figure out where best to surface this. In the meantime though, I think logging this while we can is still a good idea... It means that if/when we do surface it, the data is more likely to be available. The data is also available for debugging, should a developer need to dig into it. What do you think?

Reply:
In the spirit of keeping things simple, I've removed the capturing of the log action.